### PR TITLE
Automate blogpost imports with rss

### LINKED ISSUE #13462: Automatically import posts thru RSS feed
We currently post content to both https://paragraph.xyz/@unlockprotocol and https://unlock-protocol.com/blog
We want to avoid having to do that step twice. For this, we should have a mechanism that automatically import posts from Paragraph and adds them automatically to our git repo inside https://github.com/unlock-protocol/unlock/tree/master/unlock-protocol-com/blog

The steps to do that:
* create a Github action that runs once per day
* This action will fetch the RSS feed from Paragraph https://paragraph.xyz/api/blogs/rss/@unlockprotocol
* And for each post, it will create a [new file this folder](https://github.com/unlock-protocol/unlock/tree/master/unlock-protocol-com/blog) with the right _header_ section (`title`, `subtitle`, `authorName`, `publishDate`, `description` and `image`.
* the script should also fetch any image in the posts and store them locally in a [subfolder of this one](https://github.com/unlock-protocol/unlock/tree/master/unlock-protocol-com/public/images/blog) and replace the image URL to point to the local images (no hotlinking!)
* Once done, the script should open a PR automatically to make sure these posts are added to the git repo!

### PR DESCRIPTION
**Pull request by njokuScript**
# Description
This PR creates a GitHub action that automatically import posts thru RSS feed

# Issues

Fixes #13462 
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — COMMENTED** [review]
ok this looks good.
What is the best way to test it?

**Comment by njokuScript** [comment]
> ok this looks good. What is the best way to test it?

Was just about to ask you😅.

But I think we can merge it to the master branch, create a test blog post on paragraph that we can delete after testing to see if it creates a pr for the blog. I'll modify the cronjob to run in a minute.

**Comment by julien51** [comment]
Ha! Amother way would be to run the Github action on this branch?

File: .github/actions/blog/action.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Fetch and Create Blog Posts
+description: Fetches the latest blog posts from the Unlock Protocol blog and creates new posts in the website repository.
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run once per day at midnight
```
Comment (julien51) [inline .github/actions/blog/action.yml:5]:
```suggestion
on:
  pull_request:
      branches:
          - 'njokuScript:automate-blogpost-imports-with-rss'
  schedule:
    - cron: '0 0 * * *'  # Run once per day at midnight
```

**Comment by julien51** [comment]
ok well I guess I will merge and let's see what happens!

**Comment by njokuScript** [comment]
> ok well I guess I will merge and let's see what happens!

Alrighty!

**Pull request merged**